laravel-app/app/Services/UserVerificationService.php

```diff
@@ -6,7 +6,13 @@
 
 class UserVerificationService
 {
-    public function isUserVerified($user)
+    /**
+     * Check if the user is verified.
+     *
+     * @param mixed $user
+     * @return bool
+     */
+    public function isUserVerified($user): mixed
```

Comment:
Since they are using Laravel 10. Laravel already removed doc block and used return type declaration.

Docs:
https://laravel.com/docs/10.x/releases#types
https://www.php.net/manual/en/language.types.declarations.php

Demo:
https://www.reddit.com/r/laravel/comments/11328e0/quick_demo_of_the_laravel_10x_shift/